lib/google-apis/auth-client.js

```diff
@@ -125,7 +125,8 @@ RequestHandler.prototype.sendError = function(errorMessage, userCb) {
     cb(new Error([
       'Stackdriver error reporting client has not been configured to send',
       'errors, please check the NODE_ENV environment variable and make sure it',
-      'is set to production'
+      'is set to production or set the force property to true in the runtime',
```

Comment:
maybe decorate production and other attributes with ":
is set to "production" or set the "force" property to true in the runtime